proto/panacea/datapool/v2/pool.proto

```diff
@@ -54,7 +54,11 @@ message PoolParams {
   repeated string          data_schema = 1;             // JSON Schema URIs
   uint64                   target_num_data = 2;         // A target num of data to be collected
   uint64                   max_nft_supply = 3;          // A max num of NFTs that can be minted (= A total num of potential buyers)
-  cosmos.base.v1beta1.Coin nft_price = 4;               // A price of a NFT
```

Comment:
Is there any reason to change to accept multiple coins?

Reply:
Actually, I don't remember why I changed the type of `nft_price` to sdk.Coins. So I do roll back it